docs/docs/resources/editor_integration/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "yaml.schemas": {
-        "https://bakdata.github.io/kpops/latest/schema/config.json": "config.yaml",
```

Comment:
is there any way to get this redirection to work? that would be great!

Reply:
I will look into it